docs/Zendesk-batchsource.md

```diff
@@ -28,7 +28,7 @@ for API Token generation.
 
 **Object to Pull:** Objects to pull from Zendesk API.
 
-**Start Date:** Filter data to include only records which have Zendesk modified date is greater than 
+**Start Date:** Filter data to include only records which have Zendesk modified date is greater than
```

Comment:
Can we reword to "Filter data to include only records that have a Zendesk modified date that is greater than